config/initializers/mailer.rb

```diff
@@ -5,8 +5,8 @@
     address:              'smtp.gmail.com',
     port:                 587,
     domain:               'socius.herokuapp.com',
-    user_name:            ENV["GMAIL_USERNAME"],
-    password:             ENV["GMAIL_PASSWORD"],
+    user_name:            ENV["SENDGRID_USERNAME"],
```

Comment:
Don't use sendgrid for dev. Our sendgrid tier's capacity is limited. I've sent you our gmail info, so just use that please.